packages/libthread-db/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://android.googlesource.com/platform/ndk/
+TERMUX_PKG_DESCRIPTION="Thread debugging library"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=22
+#TERMUX_PKG_SRCURL=https://android.googlesource.com/platform/ndk/+archive/refs/tags/ndk-r${TERMUX_PKG_VERSION}/sources/android/libthread_db.tar.gz
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
```

Comment:
You can use also set `TERMUX_PKG_SHA256` to "SKIP_CHECKSUM"

Reply:
Fixed. Thanks!